bears/python/RadonBear.py

```diff
@@ -18,39 +19,56 @@ class RadonBear(LocalBear):
     CAN_DETECT = {'Complexity'}
 
     def run(self, filename, file,
-            radon_ranks_info: typed_list(str) = (),
-            radon_ranks_normal: typed_list(str) = ('C', 'D'),
-            radon_ranks_major: typed_list(str) = ('E', 'F'),
-            ):
+            cyclomatic_complexity: int=-1,
+            radon_ranks_info: typed_list(str)=(),
+            radon_ranks_normal: typed_list(str)=('C', 'D'),
+            radon_ranks_major: typed_list(str)=('E', 'F')):
```

Comment:
there has been a style change, and you are using the old style.

Reply:
I ran coala with the latest setup, didn't found any violation.